### PR TITLE
pagination added to Artist, Venues and Notes list page

### PR DESCRIPTION
**Pull request by ghost**
Pagination is added to the Artist, Venue and Note list pages

### CODE REVIEW

File: lmn/templates/lmn/artists/artist_list.html [hunk 1]
```diff
@@ -16,16 +16,47 @@ <h2 id='artist_list_title'>Artists matching '{{ search_term }}' <a href="{% url
...
 {% empty %}
-<P>No artists found</P>
+<tr><td colspan="2">No artists found</td></tr>
 {% endfor %}
+</tbody>
+</table>
+<div>
 
+{% if artists.has_other_pages %}
+<!--Pagination if page has more list than specified for display in the views-->
```
Comment (molly-clabots) [inline lmn/templates/lmn/artists/artist_list.html:41]: Good use of notes! Is it the ''' that allows sphynx to create documentation?

Comment (molly-clabots) [reply lmn/templates/lmn/artists/artist_list.html:41]: If I would have waited I would have seen farther down in your code the answer to my question.

**Review by molly-clabots — CHANGES_REQUESTED** [review]
Every thing looks good. Good use of the documentation comments. After the conflict is resolved you should be good to merge.

**Review by Di30an — COMMENTED** [review]
The pagination looks great and seems to be working correctly. I noticed that also improved the appearance with CSS, it looks great! Maybe we could add that to a new branch for issue number 20. I am fine either way. Other than that it works and looks good. Let me know if you want to change anything otherwise I will approve it.

**Pull request merged**